templates/display/video.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block main %}
+<div id='video'>
+    <video controls autoplay width="800px" height="auto">

```

Comment:
- html size attributes have no units, can we change it to "800"?
- no need for height, and "auto" isn't valid